metriken-exposition/src/util.rs

```diff
@@ -0,0 +1,35 @@
+use std::fs::File;
+use std::io::{BufRead, BufReader, Seek};
+use std::path::Path;
+
+use parquet::errors::ParquetError;
+
+use crate::snapshot::Snapshot;
+use crate::{ParquetOptions, ParquetSchema};
+
+pub fn msgpack_to_parquet(input: &Path, output: &Path) -> Result<i64, ParquetError> {
```

Comment:
Can we add a doc comment for this. It would be nice to indicate what the result means.

I don't love the use of `&Path` for the input and output. I think we want to use AsRef<Path> instead so you could use &str as arguments.

Reply:
Good call, thanks